### DISCUSSION TITLE
Does introspection of a postgres DEFAULT NOW() occur?

### DISCUSSION THREAD

**Discussion by hcharley** [prisma/prisma#1926] [Q&A] [answered]
Does introspection of postgres datasources automatically regonize a `default` value of `now()`?

**Comment by nikolasburk** [comment, marked as answer]
Hey @charlex 👋 yes it does in the latest `alpha` (which will be released as `preview025` tomorrow).

In the meantime:

```
npm install prisma2@alpha
```

**Reply by hcharley** [reply]
Nice, very cool!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
